### ISSUE TITLE
Handle string type features in the HTTP query API

### ISSUE THREAD

**Issue by bratseth** [vespa-engine/vespa#30062]
If an input is explicitly declared with type string:

```
inputs {
      query(my_feature) string  
}

```
We should allow and interpret it as a string in the HTTP query API.

**Comment by arnej27959** [comment]
https://github.com/vespa-engine/vespa/pull/30115

**Comment by arnej27959** [comment]
confusingly, the given declaration was allowed, but equivalent to
```
inputs {
      query(my_feature) double
      query(string) double
}
```

https://github.com/vespa-engine/vespa/pull/30355

**Comment by geirst** [comment]
This was fixed in Vespa 8.308.26.
